R/tm_data_table.R

```diff
@@ -127,8 +127,9 @@ tm_data_table <- function(label = "Data Table",
   )
 }
 
-
-# ui page module
+# UI page module.
```

Comment:
You do not have dots at the end of comments in this PR
https://github.com/insightsengineering/teal.modules.general/pull/653/files
Please unify
```suggestion
# UI page module
```